images/iscsi-demo-target-tgtd/README.md

```diff
@@ -79,7 +79,7 @@ pod "iscsi-demo-target-tgtd" created
 # Run a qemu instance to see if the target can be used
 # Note: This is not testing the PV or PVC, just the service and target
 # Use ctrl-a c quit to quit
-$ kubectl run --rm -it qemu-test --image=kubevirt/libvirtd -- \
+$ kubectl run --rm -it qemu-test --image=kubevirt/libvirt-kubevirt -- \
```

Comment:
This image doesn't exist anymore. kubevirt/libvirt-kubevirt was the centralized libvirt container I believe.

kubevirt/libvirt still exists though. That is the base container for virt-launcher now.